### ISSUE TITLE
FGA misses button presses

### ISSUE THREAD

**Issue by brightEcho** [Fate-Grand-Automata/FGA#964]
### Preparation

- [X] I read the [Troubleshooting Guide](https://github.com/MathewSachin/Fate-Grand-Automata/wiki/Troubleshooting)
- [X] My question isn't answered in any other issues (even the closed ones)

### Your question

I've tried running the latest FGA build on both bluestacks5 and my samsung galaxy s20 plus ultra (android version 11), and no matter what happens it always misses the button presses to select servant skills or mystic codes. I've been able to configure and use the app just fine before the NA widescreen update, so I just want to make sure I'm not going crazy here lol.

**Comment by dominionloser123** [comment]
I've run into an issue where the app runs, but fails to press in the proper areas on my phone. Pixel 4a, running Android 12.
![Screenshot_20211117-194345](https://user-images.githubusercontent.com/86737598/142336593-7fc1e073-1d1f-4d03-9372-7b4102bcac47.png)

**Comment by jomarx** [comment]
Same issue with a Galaxy Note 10. The alignment is way off.

**Comment by SeraphSephiroth** [comment]
> I've run into an issue where the app runs, but fails to press in the proper areas on my phone. Pixel 4a, running Android 12. ![Screenshot_20211117-194345](https://user-images.githubusercontent.com/86737598/142336593-7fc1e073-1d1f-4d03-9372-7b4102bcac47.png)

That black bar shouldn't be on the side. Is your game misaligned?

**Comment by Izuni** [comment]
Have the same issue with the latest build on my S9 but my game fills the entire screen
![3dN1M2c](https://user-images.githubusercontent.com/32124011/142502108-d561bf8d-1922-49ae-b732-c3e43ebeb38d.jpg)

**Comment by reconman** [comment]
@brightEcho Which version are you using? Can you provide a video?

@Izuni Does FGA see anything? Was it messed up by some screen recorder, maybe?

**Comment by reconman** [comment]
@dominionloser123 and @SeraphSephiroth see https://github.com/Fate-Grand-Automata/FGA/issues/965#issuecomment-973339177

**Comment by Izuni** [comment]
@reconman Found out the issue was caused by not extracting the default support images in the settings. Completely ignored it since it wasn't something I needed to do in the old build or I forgot about it. Thanks though!

**Comment by Wingi91** [comment]
@brightEcho check in your settings for the samsung if you can force fgo into fullscreen mode. https://github.com/Fate-Grand-Automata/FGA/wiki/Game-Area-detection#how-fgo-should-look-with-wide-screen-update

**Comment by brightEcho** [comment]
> @brightEcho check in your settings for the samsung if you can force fgo into fullscreen mode. https://github.com/Fate-Grand-Automata/FGA/wiki/Game-Area-detection#how-fgo-should-look-with-wide-screen-update

Yep, I already checked that box. And to answer your question in the other thread (since it got closed), yes, my fgo app goes completely full screen where it doesn't leave any space for the notch/camera hole.

**Comment by brightEcho** [comment]
> @brightEcho Which version are you using? Can you provide a video?
> 
> @Izuni Does FGA see anything? Was it messed up by some screen recorder, maybe?

I'm using version 1556 of FGA. And yes, I'll upload a video and pictures in a few minutes.

**Comment by brightEcho** [comment]
> @brightEcho Which version are you using? Can you provide a video?
> 
> @Izuni Does FGA see anything? Was it messed up by some screen recorder, maybe?

https://user-images.githubusercontent.com/59067507/142525454-a6cf06b7-6c90-43ea-a24f-67dcc8198c57.mp4

![Screenshot_20211118-194857_FGA (CI)](https://user-images.githubusercontent.com/59067507/142528250-415ffa9a-932f-4f9a-94e2-755f2ce3e641.jpg)
![Screenshot_20211118-194902_FGA (CI)](https://user-images.githubusercontent.com/59067507/142528310-b88b849a-fb7b-4c9e-939c-226d91b3eb9f.jpg)
![Screenshot_20211118-194841_FGA (CI)](https://user-images.githubusercontent.com/59067507/142528314-c9ee993f-1aaf-4425-ad39-9c2ec595e706.jpg)

Please let me know if you need any further information. I really want to get to the bottom of this lol

**Comment by MathewSachin** [comment]
Skill confirmation, check the troubleshooting guide.
Your problem is unrelated to the new UI update.

**Comment by john-bao-git** [comment]
> Skill confirmation, check the troubleshooting guide. Your problem is unrelated to the new UI update.

Sorry to necro, but how does this fix it? I just tried it and it worked until I had to plug-suit. How come I need the skill confirmations? Also, how can I see all the areas that FGA is going to touch? I have Debug mode on, but it doesn't show all the areas like the previous pictures in this thread.

Here is my video of a similar bug. https://www.youtube.com/watch?v=MgZ7g0ARKKo
